types/validator.yaml

```diff
@@ -1,80 +1,76 @@
-          - description: "Balance at stake in Gwei."
-      slashed:
-        type: boolean
-        description: "Was validator slashed (not longer active)."
-      activation_eligibility_epoch:
-        allOf:
-          - $ref: "./primitive.yaml#/schemas/Uint64"
-          - description: "When criteria for activation were met."
-      activation_epoch:
-        allOf:
-          - $ref: "./primitive.yaml#/schemas/Uint64"
-          - description: "Epoch when validator activated. 'FAR_FUTURE_EPOCH' if not activated"
-      exit_epoch:
-        allOf:
-          - $ref: "./primitive.yaml#/schemas/Uint64"
-          - description: "Epoch when validator exited. 'FAR_FUTURE_EPOCH' if not exited."
-      withdrawable_epoch:
-        allOf:
-          - $ref: "./primitive.yaml#/schemas/Uint64"
-          - description: "When validator can withdraw or transfer funds. 'FAR_FUTURE_EPOCH' if not defined"
+Validator:
+  type: object
+  properties:
+    pubkey:
+      $ref: './primitive.yaml#/schemas/PublicKey'
+    withdrawal_credentials:
+      allOf:
+        - $ref: "./primitive.yaml#/schemas/Root"
+        - description: "Root of withdrawal credentials"
+    effective_balance:
+      allOf:
+        - $ref: "./primitive.yaml#/schemas/Gwei"
+        - description: "Balance at stake in Gwei."
+    slashed:
+      type: boolean
+      example: false
+      description: "Was validator slashed (not longer active)."
+    activation_eligibility_epoch:
+      allOf:
+        - $ref: "./primitive.yaml#/schemas/Uint64"
+        - description: "When criteria for activation were met."
+    activation_epoch:
+      allOf:
+        - $ref: "./primitive.yaml#/schemas/Uint64"
+        - description: "Epoch when validator activated. 'FAR_FUTURE_EPOCH' if not activated"
+    exit_epoch:
+      allOf:
+        - $ref: "./primitive.yaml#/schemas/Uint64"
+        - description: "Epoch when validator exited. 'FAR_FUTURE_EPOCH' if not exited."
+    withdrawable_epoch:
+      allOf:
+        - $ref: "./primitive.yaml#/schemas/Uint64"
+        - description: "When validator can withdraw or transfer funds. 'FAR_FUTURE_EPOCH' if not defined"
 
-  AttesterDuty:
-    type: object
-    properties:
-      validator_pubkey:
-        $ref: './primitive.yaml#/schemas/PublicKey'
-      aggregator_modulo:
-        allOf:
-          - $ref: './primitive.yaml#/schemas/Uint64'
-          - description: "Value which validator can use to calculate is_aggregator (aka bytes_to_int(hash(slot_signature)[0:8]) % aggregator_modulo == 0)"
-      attestation_slot:
-        allOf:
-          - $ref: "./primitive.yaml#/schemas/Uint64"
-          - description: "The slot at which the validator must attest."
-      committee_index:
-        allOf:
-          - $ref: "./primitive.yaml#/schemas/Uint64"
-          - description: "The committee index"
+AttesterDuty:
+  type: object
+  properties:
+    validator_pubkey:
+      $ref: './primitive.yaml#/schemas/PublicKey'
+    aggregator_modulo:
```

Comment:
This doesn't match the return info on the spreadsheet.

Reply:
Haven't actually update anything. Just removed one level of tabs. Will update this in pr for /validator endpoints